packages/ses/test/error/test-aggregate-error.js

```diff
@@ -0,0 +1,46 @@
+import test from 'ava';
+import '../../index.js';
+
+const { getOwnPropertyDescriptor } = Object;
+
+lockdown();
+
+test('aggregate error', t => {
```

Comment:
do we run these tests on XS?

checking ci logs...

Reply:
I don't see any testing on XS in https://github.com/endojs/endo/actions/runs/7875562936/job/21487701027

Reply:
@kriskowal @mhofman @michaelfig @phoddie @raphdev @gibson042 

Can we set up automated endo testing on XS yet? Of even selected endo tests?

Reply:
As a temporary one-off, I changed https://github.com/Agoric/agoric-sdk/pull/7937 from `#endo-branch: master` to `#endo-branch: markm-550-any-w-aggregate-error` and rebased on "current" agoric-sdk master. CI is running now. This won't run any endo tests on XS, but it will run lots of endo on XS. If this PR breaks endo on XS, we may see it there.

Reply:
Already found a non-XS integration problem at https://github.com/Agoric/agoric-sdk/actions/runs/7877458111/job/21493876238?pr=7937#step:4:540

```
/home/runner/work/agoric-sdk/agoric-sdk/packages/agoric-cli/src/install.js
Error:     1:19  error    'AggregateError' is already defined as a built-in global variable  no-redeclare
```

on that mysterious `install.js` module. Attn @michaelfig

Reply:
Trying again without the offending global comment declaration of `AggregateError`.

Btw, when I try to `yarn lint` or `yarn lint:eslint` locally in agoric-cli I get 
```
agoric-cli(markm-test-w-some-endo-2)$ yarn lint:eslint
yarn run v1.22.19
$ eslint .

<--- Last few GCs --->

[75448:0x7fd0e8008000]    24756 ms: Mark-sweep 4061.2 (4140.6) -> 4054.5 (4142.1) MB, 668.5 / 0.0 ms  (average mu = 0.526, current mu = 0.082) allocation failure scavenge might not succeed
[75448:0x7fd0e8008000]    26058 ms: Mark-sweep 4063.4 (4142.8) -> 4056.5 (4144.3) MB, 1290.3 / 0.0 ms  (average mu = 0.259, current mu = 0.009) allocation failure scavenge might not succeed


<--- JS stacktrace --->

FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory
 1: 0x102100675 node::Abort() (.cold.1) [/Users/markmiller/.nvm/versions/node/v16.19.1/bin/node]
 2: 0x100df8f59 node::Abort() [/Users/markmiller/.nvm/versions/node/v16.19.1/bin/node]
 3: 0x100df913e node::OOMErrorHandler(char const*, bool) [/Users/markmiller/.nvm/versions/node/v16.19.1/bin/node]
 4: 0x100f6e100 v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [/Users/markmiller/.nvm/versions/node/v16.19.1/bin/node]
 5: 0x100f6e0c3 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [/Users/markmiller/.nvm/versions/node/v16.19.1/bin/node]
 6: 0x10110f815 v8::internal::Heap::FatalProcessOutOfMemory(char const*) [/Users/markmiller/.nvm/versions/node/v16.19.1/bin/node]
 7: 0x10110e19c v8::internal::Heap::CollectGarbage(v8::internal::AllocationSpace, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [/Users/markmiller/.nvm/versions/node/v16.19.1/bin/node]
 8: 0x10111aa40 v8::internal::Heap::AllocateRawWithLightRetrySlowPath(int, v8::internal::AllocationType, v8::internal::AllocationOrigin, v8::internal::AllocationAlignment) [/Users/markmiller/.nvm/versions/node/v16.19.1/bin/node]
 9: 0x10111aac1 v8::internal::Heap::AllocateRawWithRetryOrFailSlowPath(int, v8::internal::AllocationType, v8::internal::AllocationOrigin, v8::internal::AllocationAlignment) [/Users/markmiller/.nvm/versions/node/v16.19.1/bin/node]
10: 0x1010e7b57 v8::internal::Factory::NewFillerObject(int, bool, v8::internal::AllocationType, v8::internal::AllocationOrigin) [/Users/markmiller/.nvm/versions/node/v16.19.1/bin/node]
11: 0x10149f68e v8::internal::Runtime_AllocateInYoungGeneration(int, unsigned long*, v8::internal::Isolate*) [/Users/markmiller/.nvm/versions/node/v16.19.1/bin/node]
12: 0x1018483d9 Builtins_CEntry_Return1_DontSaveFPRegs_ArgvOnStack_NoBuiltinExit [/Users/markmiller/.nvm/versions/node/v16.19.1/bin/node]
13: 0x101849245 Builtins_StringAdd_CheckNone [/Users/markmiller/.nvm/versions/node/v16.19.1/bin/node]
14: 0x1119aaa16 
15: 0x111c8bb86 
error Command failed with signal "SIGABRT".
```

Reply:
With that global comment declaration gone from https://github.com/Agoric/agoric-sdk/pull/7937 , its CI succeeds with 
`#endo-branch: markm-550-any-w-aggregate-error`

That's some evidence.

Reply:
https://github.com/Agoric/agoric-sdk/pull/8898

Reply:
In light of reviewer reactions below, I'm resolving this conversation without yet having done explicit XS testing of this PR.